### DISCUSSION TITLE
folders

### DISCUSSION THREAD

**Discussion by Dixxi0509** [tahoma2d/tahoma2d#1502] [Q&A]
In tahoma2d 1.4.3 are the new folders there or is it gone forever? the folders in tahoma2d help me be organized

**Comment by manongjohn** [comment]
Folders are currently only in the Nightly versions. They will be officially available in T2D v1.5, expected to be release in Dec.

**Comment by Dixxi0509** [comment]
Am I able to get that in the nightly version of 1.4.3 when I try to get it
from github it was expired

**Reply by manongjohn** [reply]
Just download the latest nightly. It has eveything in the latest release + everything that is currently going to be in v1.5.

**Comment by Dixxi0509** [comment]
Ok thanks btw sorry that i don't know the update my discord account that I
use to get in the tahoma2d was hacked I have a new account but it feels off
when using it when it comes to discord it just doesn't feel the same
anymore
